app/services/video_service.rb

```diff
@@ -0,0 +1,24 @@
+class VideoService
+  def get_videos(category)
+    # get_json('/api/v1/videosfind_all', category)
+    response = if category == 'motivational'
+                 File.read('app/mock/motivational_video.json')
+               elsif category == 'meditation'
+                 File.read('app/mock/meditation_video.json')
+               else
+                 File.read('app/mock/yoga_video.json')
+               end
+    JSON.parse(response, symbolize_names: true)
```

Comment:
If you wanted to refactor, you could do 
`File.read("app/mock/#{category}_video.json")`

Reply:
We could but this code is just temporary before we build out the microservice.